components/profile-controller/Makefile

```diff
@@ -0,0 +1,63 @@
+
+# Image URL to use all building/pushing image targets
+IMG ?= gcr.io/kubeflow-images-public/notebook-controller
+TAG ?= $(eval TAG := $(shell date +v%Y%m%d)-$(shell git describe --tags --always --dirty)-$(shell git diff | shasum -a256 | cut -c -6))$(TAG)
+GOLANG_VERSION ?= 1.11.5
+
+all: test manager
+
+# Run tests
+test: generate fmt vet manifests
+	echo "Skip test..."
+	#go test ./pkg/... ./cmd/... -coverprofile cover.out
+
+# Build manager binary
+manager: generate fmt vet
+	go build -o bin/manager github.com/kubeflow/kubeflow/components/profile-controller/cmd/manager
+
+# Run against the configured Kubernetes cluster in ~/.kube/config
+run: generate fmt vet
+	go run ./cmd/manager/main.go
+
+# Install CRDs into a cluster
+install: manifests
+	kubectl apply -f config/crds
+
+# Deploy controller in the configured Kubernetes cluster in ~/.kube/config
+deploy: manifests
+	kubectl apply -f config/crds
+	kustomize build config/default | kubectl apply -f -
+
+# Generate manifests e.g. CRD, RBAC etc.
+manifests:
```

Comment:
this won't work since there is no vendor directory. I'll update with the correct command a bit later